app/views/works/_form_hidden_fields.html.erb

```diff
@@ -8,8 +8,6 @@
   <input type="text" id="new_title_count" name="new_title_count" value="0" class="hidden" />
   <input type="text" id="creator_count" name="creator_count" value="<%= @work&.resource&.creators&.count || 0%>" class="hidden" />
   <input type="text" id="contributor_count" name="contributor_count" value="<%= @work&.resource&.contributors&.count || 0%>" class="hidden" />
-  
-  <!-- The count tells the controller how many related objects to expect -->
```

Comment:
Why remove this comment.  It still seems relevant

Reply:
The same hold for all the fields above: `new_title_count` tells the controller how many new titles to expect, `creator_count` tells the controller how many creators to expect, etc. 

In the slightly longer run, we won't need these fields as we move towards using the `[]` rails convention for repeated field names.